### ISSUE TITLE
GEM: Jaw readback incorrect

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#5940]
As a scientists on GEM I would like to be able to move my jaws correctly.

## Acceptance Criteria
* Jaws move to setpoints correctly and are able to do the full motion of travel

**Comment by DominicOram** [comment]
Issue seems to have been caused by the network switch connecting the Galil to the NDX having failed. Electronics got this running again (unsure if this was by replacing or restarting the switch). After this some axes were hitting hard limits in unexpected places, implying they had lost their positions. The jaws manager also seemed to be in a state where it was reporting incorrect values, likely caused by the failed network switch.

I solved this by:
* Restarting all galil IOCs
* Rehoming all axes

Follow up actions:
* I found a bug when homing some of the axes (https://github.com/ISISComputingGroup/IBEX/issues/5941)
* A number of tickets have been proposed that would make the failing network switch more obvious (https://github.com/ISISComputingGroup/IBEX/issues/4745, https://github.com/ISISComputingGroup/IBEX/issues/3521)

**Comment by davidkeymer** [comment]
Have since discovered that the RCD supplying power to the network switch had tripped for an unknown reason.  This was simply reset to resolve the communications issue.  The problem was not caused by the failure of the switch itself in this case, although it would still be worth checking that the model in use is a more reliable gigabit capable unit.

**Comment by DominicOram** [comment]
Yes, it is a gigabit switch, I checked the other day.

**Comment by DominicOram** [comment]
Also added https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Motors-Trouble-Shooting#motors-are-hitting-a-hard-limit-in-an-unexpected-place
